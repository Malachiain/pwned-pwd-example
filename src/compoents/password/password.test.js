import React, {Component} from 'react';
import {shallow} from 'enzyme';
import toJson from 'enzyme-to-json';
import {Password} from './index'
describe('The pasword entry screen',()=>{
    it('It renders the password screen correctly',()=>{
        const wrapper = shallow(<Password password="abc123" updatePassword={()=>{} } checkStatus={()=>{} } />)
        expect(toJson(wrapper)).toMatchSnapshot();
    })
})