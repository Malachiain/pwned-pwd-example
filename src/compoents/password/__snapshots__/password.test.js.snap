// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The pasword entry screen It renders the password screen correctly 1`] = `
<styled.div>
  <styled.div>
    <styled.div>
      <styled.label
        htmlFor="password"
      >
        Password
      </styled.label>
      <styled.input
        id="password"
        onChange={[Function]}
        type="password"
        value="abc123"
      />
      <styled.button
        onClick={[Function]}
      >
        Check password
      </styled.button>
    </styled.div>
    <styled.div>
      <styled.p>
        Enter a password that you want to check. The password will not be recorded. Only the first 5 characters of the sha1 hash of your password will be sent to the API. Trusting people on the internet is a dangerous game, so it is problably not a good idea to a enter a real password you use.
      </styled.p>
    </styled.div>
  </styled.div>
</styled.div>
`;
